### DISCUSSION TITLE
APK本地化相关

### DISCUSSION THREAD

**Discussion by chihya72** [chinosk6/gakuen-imas-localify#37] [General]
请问作者后续有考虑解包APK内的octodb然后加上本地化钩子吗，貌似成就说明、技能饮料介绍这些都是在APK内的，这些我也想尝试本地化

**Comment by chinosk6** [comment]
能详细说一下吗，弄清楚之后肯定会做这一部分。现在还有一部分文本未找到 hook 点，比如支援卡相关文本，还有你提到的部分等

**Comment by chihya72** [comment]
![QQ截图20240602002342](https://github.com/chinosk6/gakuen-imas-localify/assets/29171277/f6aac798-21a9-45c7-931a-2729cb56e1c9)
octo文件夹内有一个l文件，里面有23行哈希值，刚好octo内有23个文件，我个人认为应该哈希值就是这些文件的解密密钥，我按照解密资源包的方式直接对每个文件进行解密但是失败了
